HoloPipelines/server.py

```diff
@@ -0,0 +1,74 @@
+"""
+This module represents the entrypoint of the HoloPipelines. It starts
+a Flask server and listens to endpoints, allowing external clients to
+start jobs and trace their progress.
+"""
+
+import logging
+
+import coloredlogs
+from flask import Flask, request
+from flask_json import as_json
+
+from config import FLASK_ENV, APP_PORT
+from core.pipelines.pipelines_controller import get_pipelines_dict
+from jobs import jobs_controller
+from jobs.jobs_state import activate_periodic_garbage_collection, get_current_state
+from jobs.jobs_io import read_log_file_for_job
+
+log_format = "%(asctime)s | %(name)s | %(levelname)s | %(message)s'"
+coloredlogs.install(level=logging.DEBUG, fmt=log_format)
```

Comment:
Log level hard coded to DEBUG might not be ideal when switched to production mode. Could be tied i
```suggestion
log_level = logging.DEBUG if FLASK_ENV == "development" else logging.INFO
coloredlogs.install(level=log_level, fmt=log_format)
```